### DISCUSSION TITLE
VS - Code Snippet

### DISCUSSION THREAD

**Discussion by ekallaur** [ardalis/GuardClauses#151] [Show and tell]
Simple VS - Code Snippet: https://gist.github.com/ekallaur/36f1f32b07fa694b5a75d929e2e70de9
Or there're already a lot of high-quality snippets but I missed them?

**Comment by ardalis** [comment]
Nice. I wonder if anyone would download a VS Code extension that simply consisted of snippets for various Ardalis.* packages...

**Reply by ekallaur** [reply]
I have no answer for that one. We are using Ardalis.GuardClause in the projects and what I'm sure about is that this simple code snippet saves a tone of time during development.
